samples/peripheral/802154_phy_test/prj.conf

```diff
@@ -16,8 +16,8 @@ CONFIG_NRFX_POWER=y
 
 # Enable RNG
 CONFIG_NRFX_RNG=y
-# Use XOROSHIRO PRNG
-CONFIG_XOROSHIRO_RANDOM_GENERATOR=y
```

Comment:
What's the reason for this change?

Reply:
https://docs.zephyrproject.org/latest/releases/release-notes-3.5.html

> CONFIG_XOROSHIRO_RANDOM_GENERATOR, deprecated a long time ago, is finally removed.